docs/sources/docker-hub/official_repos.md

```diff
@@ -1,189 +1,113 @@
-page_title: Guidelines for Official Repositories on Docker Hub
+page_title: Official Repositories on Docker Hub
 page_description: Guidelines for Official Repositories on Docker Hub
 page_keywords: Docker, docker, registry, accounts, plans, Dockerfile, Docker Hub, docs, official, image, documentation
 
-# Guidelines for Creating and Documenting Official Repositories
+# Official Repositories on Docker Hub
+
+You should know by now that a Docker repository (repo) is a collection of
```

Comment:
I just pushed a new version that removes this initial reminder.  I had tried to squeeze it into the first sentence, then moxie improved the wording, but now I think the first sentence just needs to get straight to the point.  One of the previous sections of the Docker Hub tab talks about repositories and images, and for users that don't really know the difference, they can still understand the page.